aiscripts/supply_mule.xml

```diff
@@ -5113,11 +5116,11 @@
 							<!-- if we end up with more than one need, we'll move it down with move_order later -->
 							<do_if value="$bestProfit gt 0">
 								<do_if value="$bestSupplyTrade.available and $bestNeedTrade.available">
-									<debug_to_file chance="$debugchance" name="'SupplyMule- ' + this.ship.knownname + ' ' + this.ship.idcode" directory="'MulesExtended'" text="'buying ' +$bestAmount + ' ' +$bestNeedTrade.ware +' from ' +$bestSupplyTrade.owner.knownname
+									<debug_to_file chance="$debugchance" name="$debugFileName" directory="'MulesExtended'" text="'buying ' +$bestAmount + ' ' +$bestNeedTrade.ware +' from ' +$bestSupplyTrade.owner.knownname
 									+' at ' +$bestSupplyTrade.unitprice.formatted.{'%s %Cr'} + ' to sell to ' +$bestNeedTrade.owner.knownname
 									+' at ' +$bestNeedTrade.unitprice.formatted.{'%s %Cr'} + ' for a profit of ' +$bestProfit.formatted.{'%s %Cr'} " />
 	
-									<write_to_logbook category="upkeep" title="'SupplyMule- ' + this.ship.knownname + ' ' + this.ship.idcode" interaction="showonmap" money="$bestProfit" text="'buying ' +$bestAmount + ' ' +$bestNeedTrade.ware +' from ' +$bestSupplyTrade.owner.knownname
+									<write_to_logbook category="upkeep" title="$debugFileName" interaction="showonmap" money="$bestProfit" text="'buying ' +$bestAmount + ' ' +$bestNeedTrade.ware +' from ' +$bestSupplyTrade.owner.knownname
```

Comment:
We may actually want the ship name in the logbook entries.

Reply:
Ah, mass replace... Let me check that

Reply:
should be seperated now, see commit https://github.com/Misunderstood-Wookiee/Mules-and-Warehouses-Extended/pull/47/commits/d30d289c8f7b4d402fc69a274b3163de21e18b5e